apps/research-service/src/site-polygons/site-polygon-query.builder.ts

```diff
@@ -53,7 +53,18 @@ export class SitePolygonQueryBuilder extends PaginatedQueryBuilder<SitePolygon>
   constructor(pageSize?: number) {
     super(SitePolygon, pageSize);
 
-    this.findOptions.include = [{ model: PolygonGeometry, attributes: ["polygon"], required: true }, this.siteJoin];
+    this.findOptions.include = [
+      {
+        model: PolygonGeometry,
+        attributes: [
+          "polygon",
+          [literal("ST_Y(ST_Centroid(geom))"), "centroidLatitude"],
+          [literal("ST_X(ST_Centroid(geom))"), "centroidLongitude"]
+        ],
+        required: true
```

Comment:
This is going to mean that if a site doesn't have an associated polygon, it won't be returned at all. Is that the desired behavior?